R/use.R

```diff
@@ -96,7 +96,8 @@ use <- function(...,
 }
 
 renv_use_libpath <- function() {
-  global("use.library", tempfile("renv-library-"))
+  the$use.library <- the$use.library %||% tempfile("renv-library-")
+  the$use.library
```

Comment:
This line might not be necessary? (The above should invisibly return the assigned value IIUC)